newIDE/app/src/ResourcesLoader/index.js

```diff
@@ -146,4 +147,22 @@ export default class ResourcesLoader {
 
     return resourceName;
   }
+
+  /**
+   * Get the fully qualified URL/filename associated with the given resource.
```

Comment:
When copying stuff, prefer to leave out comments rather than putting comments that are not correct, because it make a lot more work when reviewing as I have to proofreader everything.

Reply:
ah sorry, updated it now